app/assets/src/stylesheets/02-molecules/forms.scss

```diff
@@ -7,6 +36,23 @@ input[type=checkbox] + label {
   margin-bottom: 0;
 }
 
-.field, form > .actions {
+.field form > .actions {
   margin: 1.5em auto;
-}
\ No newline at end of file
+}
+
+.actions {
```

Comment:
If you're renaming one class to BEM-style instead of using `>` nesting to get past a name collision, then please rename all of them. We get maintainability & onboardability through being conscious about what patterns we reify as exemplars and which ones we don't, and this seems a bit more like "it was annoying to rename both so I only renamed one." This is blocking.

Reply:
I'm definitely not attached to any particular implementation, I'm just floundering trying to make the UI do the thing I want it to. If you have a preference for something in particular please state it and give an example or link so I don't have to spend time going down rabbit holes that don't bring my tactics in line with the tactics you want.

Reply:
Okay! Rephrasing more concretely: what you seem to be doing here is defining the `.actions` that are in a `.project-summary` as different than the `.actions` that are in a `form`. That is fine! It seems like they are shaping up to be different things, even though I thought at first they might be similar.

Back when I thought that the two different `.actions` molecules might be the same molecule, I was using the `.actions` name for both and then changing how they were styled in the context of the larger "project summary" and "form" molecules/organisms/whatever using `>` nesting -- which I wasn't in love with but which worked to push stuff forward. And it seems like you're renaming `.project-summary__actions` to disambiguate the two. This is totally fine, but because you're also bumping form `.actions` out from underneath `form >` all of a sudden we have a situation in which the fact that `.actions` now only belong inside `form`s is near-totally implicit -- like, they're in the `forms.scss` file, which helps, but I'm not going to immediately see that when I open up the web inspector in Firefox.

So what I'm thinking is that we can EITHER decide that action bars are container-agnostic (and revert to the old `.actions` for both forms and project summaries) OR we can decide that they are container dependent and rename BOTH different kinds of action bars to `form.__actions` and `.project-summary__actions`. I don't care which one we do, but I'd like us to pick one or the other instead of straddling the line. Does that make more sense?